schema.md

```diff
@@ -23,10 +23,9 @@ name             | string    | not null
 pick/ban rate    | integer   |
 win rate         | integer   |
 metascore        | integer   |
-top              | integer   | not null
-mid              | integer   | not null
-jungle           | integer   | not null
-marksman         | integer   | not null
-support          | integer   | not null
-
+role             | string    | must be between 00000 and 11111
 image            | img       | not null
+
+NB: `role` above is a binary string with digits ordered by top-mid-jungle-marksman-support,
```

Comment:
Role will be easier to check against if it's an integer within the range 0b11111 to 0b00000 with bitwise operators.